lib/registries/addon/branded/route.ts

```diff
@@ -2,15 +2,25 @@ import { action } from '@ember/object';
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 import DS from 'ember-data';
+import RSVP from 'rsvp';
 
 import Analytics from 'ember-osf-web/services/analytics';
+import Brand from 'registries/services/brand';
 
 export default class BrandedRegistriesRoute extends Route {
     @service analytics!: Analytics;
     @service store!: DS.Store;
+    @service brand!: Brand;
 
     model(params: { providerId: string }) {
-        return this.store.findRecord('registration-provider', params.providerId);
+        return RSVP.hash({
+            registrationProvider: this.store.findRecord('registration-provider', params.providerId),
+            brand: this.store.findRecord('brand', '1'), // Maybe not the right way to fetch?
```

Comment:
What is the right way to fetch a brand? I'm assuming we'll get it from the registration provider?

Reply:
you should be able to embed/include the brand in the prior request:
```
this.store.findRecord('registration-provider', params.providerId, { include: 'brand' });
```